bot.py

```diff
@@ -37,10 +37,17 @@
 STARBOARD_MESSAGE_ID = None
 TITLE = None
 
+
+GUILDS=[discord.Object(id=1137183815949881357), discord.Object(id=402891511991369740)]
```

Comment:
just for DX, is it possible for the bot to discover what servers its in on it's own?